docs/getting-started.md

```diff
@@ -360,10 +371,9 @@ describe('#isString()', () => {
 });
 ```
 
-* note that the `__impl__` is a reference to the impl file injected by the testing environment.
-* you can't use node modules like you would normal do, because the component should be exported to an isolated environment and run the specs there. you can only require the modules that the tester provides,
+* you can't use node modules like you would normal do, because the component should be exported to an isolated environment and run the specs there. you can only require the modules that the tester provides.
```

Comment:
> you can't use node modules like you would normal do, because the component should be exported to an isolated environment and run the specs there. you can only require the modules that the tester provides.

normally do